gems/pending/postgres_ha_admin/failover_monitor.rb

```diff
@@ -10,15 +10,20 @@ class FailoverMonitor
     FAILOVER_ATTEMPTS = 10
     DB_CHECK_FREQUENCY = 300
     FAILOVER_CHECK_FREQUENCY = 60
+    LOG_FILE = '/var/www/miq/vmdb/log/ha_admin.log'.freeze
     attr_accessor :failover_attempts, :db_check_frequency, :failover_check_frequency
 
     def initialize(db_yml_file: '/var/www/miq/vmdb/config/database.yml',
                    failover_yml_file: '/var/www/miq/vmdb/config/failover_databases.yml',
                    ha_admin_yml_file: '/var/www/miq/vmdb/config/ha_admin.yml',
-                   log_file: '/var/www/miq/vmdb/log/ha_admin.log',
+                   logger: nil,
```

Comment:
Comment for a separate PR.

Would it make sense to use keyword args for this initializer:
Instead of
```ruby
FailoverMonitor.new("/var/www/miq/vmdb/config/my_database.yml", "/var/www/miq/vmdb/config/my_failover_dbs.yml", "/var/www/miq/vmdb/config/ha_admin.yml", nil, "test")
```
one would specify:
```ruby
FailoverMonitor.new(
  :database_config_file => "/var/www/miq/vmdb/config/my_database.yml",
  :failover_config_file => "/var/www/miq/vmdb/config/my_failover_dbs.yml",
  :ha_admin_config_file => "/var/www/miq/vmdb/config/ha_admin.yml",
  :logger => nil,
  :environment => "test")
```
which would allow making some or all of these optional:
```ruby
FailoverMonitor.new(
  :database_config_file => "/var/www/miq/vmdb/config/my_database.yml",
  :failover_config_file => "/var/www/miq/vmdb/config/my_failover_dbs.yml",
  :environment => "test")
```
or even:
```ruby
FailoverMonitor.new
```